posthog/queries/session_recordings/test/__snapshots__/test_session_recording_list_from_session_replay.ambr

```diff
@@ -1,3 +1,698 @@
+# name: TestClickhouseSessionRecordingsListFromSessionReplay.test_action_filter
+  '
+  with events_session_ids AS
+    (SELECT distinct `$session_id` as session_id
+     FROM events PREWHERE team_id = 2
+     AND timestamp >= '2022-12-27 12:00:00'
+     AND timestamp <= '2023-01-04 12:00:00'
+     AND event IN ['custom-event']
+     AND ((event = 'custom-event'
```

Comment:
huh, for future consideration, this has the same problem, everything smushed in the prewhere, including fat json columns.